gpii/node_modules/packagekit/nodepackagekit/nodepackagekit.cc

```diff
@@ -13,14 +13,22 @@ You may obtain a copy of the License at
 */
 #define I_KNOW_THE_PACKAGEKIT_GLIB2_API_IS_SUBJECT_TO_CHANGE 1
 
-#include <node.h>
-#include <v8.h>
+#include <nan.h>
 #include <packagekit-glib2/packagekit.h>
 #include <gio/gio.h>
 #include <json-glib/json-glib.h>
 #include <string>
 
 using namespace v8;
+using v8::FunctionTemplate;
```

Comment:
I'd prefer to do without these using directives so we can be crystal clear when we are using a v8 primitive and when a Nan one